src/Jupyter/Visualization/StateDisplayEncoders.cs

```diff
@@ -412,9 +412,10 @@ public StateVectorToHtmlResultEncoder(IConfigurationSource configurationSource)
                         </tbody>
                     </table>";
                 
-                if (ConfigurationSource.MeasurementDisplayHistogram) {
-                    outputTable += $@"<div id=""{vector.DivId}""></div>";
-                };
+                // TODO(rokuzmin): Temporarily commenting out to check the build.
```

Comment:
Agreed with this for now, and that we should restore this before merging to main.